src/devices/Ina226/Ina226.sln

```diff
@@ -0,0 +1,70 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio 15
+VisualStudioVersion = 15.0.26124.0
+MinimumVisualStudioVersion = 15.0.26124.0
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Ina226", "Ina226.csproj", "{B82C190A-642B-465B-BD3F-DB56FFF22253}"
+EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "tests", "tests", "{03107CFF-FE00-467D-9B38-C00F7E12A7FE}"
+EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Ina226_Not_Required.Tests", "tests\Ina226_Not_Required.Tests.csproj", "{2B0F5D91-F38B-427D-84DB-5656D9661E1C}"
```

Comment:
You haven't included any unit tests (which is not optimal, but accepted), but then you should not include this project reference.

Reply:
I have not gotten far enough in my c# knowledge yet to write tests, it's on my to-learn list up near the top. I have removed the reference to the tests project though since I will likely not get to writing tests for this very soon.